client/src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,75 @@
+import Typography from '@material-ui/core/Typography';
+import Avatar from '@material-ui/core/Avatar';
+import Divider from '@material-ui/core/Divider';
+import RoomIcon from '@material-ui/icons/Room';
+import Rating from '@material-ui/lab/Rating';
+import Paper from '@material-ui/core/Paper';
+import Grid from '@material-ui/core/Grid';
+import Link from '@material-ui/core/Link';
+
+import React, {useState, useEffect } from 'react';
+import useStyles from './useStyles';
+
+import {Profile} from "../../interface/Profile";
+
+interface CardProps {
+    profile: Profile;
+    key?: string

```

Comment:
We don't need a key field here as we are specifying a key to the parent Grid item.